.changeset/swift-socks-doubt.md

```diff
@@ -0,0 +1,9 @@
+---
+'@solana/transaction-messages': minor
+'@solana/transactions': minor
+'@solana/signers': minor
+'@solana/errors': minor
+'@solana/kit': minor
```

Comment:
This is the first in the stack that requires a `major` bump, because of the renamings, right?

```suggestion
'@solana/transaction-messages': major
'@solana/transactions': major
'@solana/signers': major
'@solana/errors': major
'@solana/kit': major
```